src/plugins/commander/commands/http.js

```diff
@@ -0,0 +1,37 @@
+const Command = require('../structs/Command');
+const { MessageAttachment } = require('discord.js');
+
+class HTTPCommand extends Command {
+    constructor(bot) {
+        super(bot);
+        this.aliases = ['http', 'catus'];
+
+        this.shortdesc = `Sends a catus code image.`;
+        this.desc = `Sends a HTTP error's cat error code image from http.cat`;
+        this.usages = [
+            '!http 404'
+        ];
+
+        this.validCodes = [
+            100, 101, 102, 103, 200, 202, 203, 204, 205, 206, 207, 300,
+            301, 302, 303, 304, 305, 306, 307, 400, 401, 402, 403, 404, 405, 406, 407,
+            408, 409, 410, 411, 412, 413, 414, 415, 416, 417, 418, 422, 423, 424, 425,
+            426, 449, 450, 499, 500, 501, 502, 503, 504, 505, 506, 507, 508, 509, 510
+        ];
+    }
+
+    async run(message, content) {
+        if (!content || !parseInt(content)) return;
+
+        if (this.validCodes.includes(parseInt(content))) {
+            const attachment = new MessageAttachment(`https://http.cat/${content}.jpg`, `${content}.jpg`)
+
+            await message.channel.send(`<https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/${content}>`, attachment);
```

Comment:
Please store the result of parseInt(content) and store that in a variable, then use it in every place that you use content in

Currently it leaks the rest of the content if you do `!http 404 more message content here`